media.css

```diff
@@ -0,0 +1,600 @@
+  .catalog-item__red,
+  .catalog-item__orange,
+  .catalog-item__brown,
+  .catalog-item__cherry,
+  .catalog-item__blue {
+    height: 20px;
+    width: 20px;
+  }
+
+  /* .video-section {
+    height: 490px;
+    background-color: #000000;
+    display: flex;
+    flex-direction: column;
+    justify-content: center;
+    padding-left: 30px;
+    margin-bottom: 117px;
+    position: relative;
+    overflow: hidden;
+    -webkit-transform: rotate(2.6deg);
+    transform: rotate(2.6deg);
+  }
+  .video-section:before {
+    content: "";
+    position: absolute;
+    width: 200%;
+    height: 200%;
+    top: -50%;
+    left: -50%;
+    z-index: -1;
+    background-color: #000;
+    -webkit-transform: rotate(-2.6deg);
+    transform: rotate(-2.6deg);
+    }
+  .video-section__image {
+    background-image: url(img/video.png);
+    display: block;
+    padding-left: 30px;
+  }
+
+  .video-section__content {
+    display: block;
+    width: 258px;
+    padding-top: 138px;
+    margin-left: 46px;
+  }
+
+  .video-content__h3 {
+    font-size: 22px;
+    line-height: 27px;
+    margin-bottom: 12px;
+    color: #ffffff;
+    text-transform: uppercase;
+  }
+
+  .video-content__description {
+    width: 248px;
+    font-size: 13px;
+    line-height: 16px;
+    margin-bottom: 10px;
+    color: #fff;
+  } */
+}
+
+@media (min-width: 375px) and (max-width: 767px) {
+  .wrapper {
+    width: 100%;
+  }
+
+  .header {
+    grid-template-columns: 1fr 1fr;
+
+    /* margin-bottom: 33px;
+    padding-top: 24px; */
+    height: 22px;
+    padding-bottom: 26px;
+    margin-bottom: 0;
+  }
+
+  .header__menu,
+  .header__search,
+  .header__user-block,
+  .header__cart,
+  .header__language-block {
+    display: none;
+  }
+
+  .header__logo-image {
+    display: none;
+  }
+
+  .header__logo-image_adaptive {
+    width: 104px;
+    height: 22px;
+    display: flex;
+    align-items: center;
+    margin-left: 48px;
+  }
+
+  .header__menu-bar {
+    display: flex;
+    justify-content: flex-end;
+    align-items: center;
+  }
+
+  .menu-bar {
+    display: block;
+    width: 41px;
+    height: 17px;
+    margin-right: 47px;
+  }
+
+  .top-content {
+    background-image: url(img/header-img-375.png);
+    background-repeat: no-repeat;
+    background-size: cover;
+    width: 100%;
+    height: 191px;
+    margin-bottom: 38px;
+  }
+
+  .top-content__block {
+    padding-top: 66px;
+    padding-left: 48px;
+  }
+
+  .top-content__h1 {
+    font-size: 20px;
+    line-height: 19px;
+    margin-bottom: 9px;
+  }
+
+  .link {
+    font-size: 12px;
+    line-height: 11px;
+  }
+
+  .hidden-375 {
+    display: none;
+  }
+
+  .catalog {
+    margin-left: 48px;
+    margin-right: 46px;
+  }
+
+  .catalog__title {
+    font-size: 20px;
+    line-height: 19px;
+    margin-bottom: 0;
+  }
+
+  .catalog-item__description {
+    display: grid;
+    grid-template-columns: 1.5fr 1fr;
+    padding-left: 11px;
+  }
+
+  .catalog__items {
+  	width: 100%;
+  	display: flex;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Expected indentation of 4 spaces (indentation)](https://app.codacy.com/gh/WebHeroSchool/PolinaYartseva_axion/pullRequest?prid=5569682)